R/cnd-abort.R

```diff
@@ -1,27 +1,88 @@
 #' Signal an error, warning, or message
 #'
 #' @description
-#'
 #' These functions are equivalent to base functions [base::stop()],
-#' [base::warning()], and [base::message()], but make it easy to supply
-#' condition metadata:
+#' [base::warning()], and [base::message()]. They signal a condition
+#' (an error, warning, or message respectively) and make it easy to
+#' supply condition metadata:
 #'
-#' * Supply `class` to create a classed condition. Typed
-#'   conditions can be captured or handled selectively, allowing for
-#'   finer-grained error handling.
+#' * Supply `class` to create a classed condition that can be caught
+#'   or handled selectively, allowing for finer-grained error
+#'   handling.
 #'
-#' * Supply metadata with named `...` arguments. This data will be
-#'   stored in the condition object and can be examined by handlers.
+#' * Supply metadata with named `...` arguments. This data is stored
+#'   in the condition object and can be examined by handlers.
 #'
-#' `abort()` throws subclassed errors, see
-#' [`"rlang_error"`][rlang_error].
+#' * Supply `call` to inform users about which function the error
+#'   occurred in.
 #'
-#' `interrupt()` allows R code to simulate a user interrupt of the
-#' kind that is signalled with `Ctrl-C`. It is currently not possible
-#' to create custom interrupt condition objects.
+#' * Supply another condition as `parent` to create a chained
+#'   condition.
```

Comment:
Might need to link to where to get more info on this?

Reply:
yup I'm planning to write this documentation in the coming days.